src/components/schemas/statement_mappings.yaml

```diff
@@ -0,0 +1,81 @@
+suggest_out:
+  type: object
+  additionalProperties: False
+  properties:
+    id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: stmpwownwrng
+    bank_name:
+      type: string
+      description: |
+        Specifies the bank name of the statement.
+      example: American Express
+    is_debit_positive:
+      type: boolean
+      description: |
+        Specifies the representation of the the amount in the statement, if true then the amount represented by 
+        negative sign are credits, and if false then the amount represented by negative sign are debits.
```

Comment:
p1 | Pls to inverse the statement, as the column name and the description is inverse and can cause strain on reader.
positive sign are debits, and if false then the amount represented by negative sign are debits.